wsd/ProxyProtocol.hpp

```diff
@@ -40,10 +40,6 @@ class ProxyProtocolHandler : public ProtocolHandlerInterface
     int getPollEvents(std::chrono::steady_clock::time_point /* now */,
                       int64_t &/* timeoutMaxMs */) override;
 
-    void checkTimeout(std::chrono::steady_clock::time_point /* now */) override
```

Comment:
Could it be that the author of this intentionally wanted to suppress a default behavior?

I say this because testing the proxy is not commonly done. And unless I can test the proxy, I wouldn't touch it.

Reply:
ProxyProtocolHandler -> ProtocolHandlerInterface, and the default is a NOP with return false.